src/js/themes/media-theme-demuxed-2022.js

```diff
@@ -0,0 +1,566 @@
+    <media-play-button>
+      <svg slot="play" viewBox="0 0 16 16">
+        <g transform="translate(-6, -6)">
+          <path d="M19.6,13.2L11.1,9c-0.6-0.3-1.2,0.1-1.2,0.7v8.5c0,0.6,0.7,1,1.2,0.7l8.5-4.2C20.2,14.4,20.2,13.6,19.6,13.2z"/>
+        </g>
+      </svg>
+      <svg slot="pause" viewBox="0 0 16 16">
+        <g transform="translate(-6, -6)">
+          <g>
+            <path d="M17.8,20c-0.5,0-0.9-0.4-0.9-0.9V8.9c0-0.5,0.4-0.9,0.9-0.9s0.9,0.4,0.9,0.9v10.2C18.7,19.6,18.3,20,17.8,20z"/>
+          </g>
+          <g>
+            <path d="M10.1,20c-0.5,0-0.9-0.4-0.9-0.9V8.9C9.2,8.4,9.6,8,10.1,8S11,8.4,11,8.9v10.2C10.9,19.6,10.5,20,10.1,20z"/>
+          </g>
+        </g>
+      </svg>
+    </media-play-button>
+  </div>
+  <media-control-bar>
+    <media-play-button class="small-button">
+      <svg slot="play" viewBox="0 0 16 16">
+        <g transform="translate(-6, -6)">
+          <path d="M19.6,13.2L11.1,9c-0.6-0.3-1.2,0.1-1.2,0.7v8.5c0,0.6,0.7,1,1.2,0.7l8.5-4.2C20.2,14.4,20.2,13.6,19.6,13.2z"/>
+        </g>
+      </svg>
+      <svg slot="pause" viewBox="0 0 16 16">
+        <g transform="translate(-6, -6)">
+          <g>
+            <path d="M17.8,20c-0.5,0-0.9-0.4-0.9-0.9V8.9c0-0.5,0.4-0.9,0.9-0.9s0.9,0.4,0.9,0.9v10.2C18.7,19.6,18.3,20,17.8,20z"/>
+          </g>
+          <g>
+            <path d="M10.1,20c-0.5,0-0.9-0.4-0.9-0.9V8.9C9.2,8.4,9.6,8,10.1,8S11,8.4,11,8.9v10.2C10.9,19.6,10.5,20,10.1,20z"/>
+          </g>
+        </g>
+      </svg>
+    </media-play-button>
+    <div class="media-volume-wrapper">
+      <media-mute-button class="small-button">
+        <svg slot="off" viewBox="0 0 16 16">
+          <g transform="translate(-6, -6)">
+            <path d="M13,8.2l-2.8,2.9v0.1H7.4c-0.5,0-0.9,0.4-0.9,0.9V16c0,0.5,0.4,0.9,0.9,0.9h2.8l2.8,2.9c0.3,0.3,0.8,0.1,0.8-0.3v-11
+          C13.8,8.1,13.3,7.9,13,8.2z"/>
+            <path d="M19.4,14l2-2c0.3-0.3,0.3-0.6,0-0.9c-0.3-0.3-0.6-0.3-0.9,0l-2,2l-2-2c-0.3-0.3-0.6-0.3-0.9,0c-0.3,0.3-0.3,0.6,0,0.9l2,2
+          l-2,2c-0.3,0.3-0.3,0.6,0,0.9c0.1,0.1,0.3,0.2,0.4,0.2c0.2,0,0.3-0.1,0.4-0.2l2-2l2,2c0.1,0.1,0.3,0.2,0.4,0.2
+          c0.2,0,0.3-0.1,0.4-0.2c0.3-0.3,0.3-0.6,0-0.9L19.4,14z"/>
+          </g>
+        </svg>
+        <svg slot="low" viewBox="0 0 16 16">
+          <g transform="translate(-6, -6)">
+            <g id="_x2D_e-speaker_00000056397674509617663590000017246565507478500284_">
+              <path d="M13.1,8.2l-2.8,3l0,0H7.5c-0.5,0-0.9,0.4-0.9,0.9V16c0,0.5,0.4,0.9,0.9,0.9h2.8l2.8,2.9c0.3,0.3,0.8,0.1,0.8-0.3v-11
+                C13.9,8.1,13.4,7.9,13.1,8.2z"/>
+            </g>
+            <g>
+              <g>
+                <path d="M16.3,17.4c-0.2,0-0.3-0.1-0.4-0.2c-0.3-0.3-0.3-0.6,0-0.9c0.6-0.6,1-1.4,1-2.3s-0.4-1.6-1-2.3c-0.3-0.3-0.3-0.6,0-0.9
+                  c0.3-0.3,0.6-0.3,0.9,0c0.9,0.8,1.4,2,1.4,3.2s-0.5,2.3-1.4,3.2C16.6,17.4,16.4,17.4,16.3,17.4z"/>
+              </g>
+            </g>
+          </g>
+        </svg>
```

Comment:
these SVG's look like they can be optimized. have you tried running them through a tool like SVGO?

Reply:
Great suggestion. I've just run all icons through SVGO and it saves quite some bytes (40% on average).